packages/tfm-ror51-rubygem-concurrent-ruby/tfm-ror51-rubygem-concurrent-ruby.spec

```diff
@@ -18,6 +18,8 @@ BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildRequires: %{?scl_prefix_ruby}ruby >= 1.9.3
 BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+Obsoletes: tfm-rubygem-%{gem_name}
```

Comment:
Usually we obsolete older versions, so `< 1.0.5-3`. That way we can later reintroduce the package in our SCL when needed.